.gitignore

```diff
@@ -12,4 +12,10 @@ yalc.lock
 coverage/
 __generated__
 
-*.tsbuildinfo
\ No newline at end of file
+*.tsbuildinfo
+
+# Barrel files
+packages/*/*
+!packages/*/package.json
+!packages/*/src
+!pacakges/*/README.md
```

Comment:
Could this work?

```suggestion
# Barrel files
## Ignore all folders
packages/*/*
## Don't ignore files
!packages/*/*.*
## Don't ignore `src` folder
!packages/*/src
```

Reply:
Yep that works. Also included `test` which was missing 👍🏼